report_thesis/src/sections/problem_definition.tex

```diff
@@ -20,71 +32,97 @@ \section{Problem Definition}\label{sec:problem_definition}
 
-We evaluate the robustness of our models using the sample standard deviation of prediction errors, defined as:
+Robustness is evaluated using the sample standard deviation of prediction errors:
 \[
 \sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
 \]
 where \( e_i = \mathbf{v}_i - \hat{\mathbf{v}}_i \) and \( \bar{e} \) is the mean error.
-Sample standard deviation is used because it provides an unbiased estimate of the variability in prediction errors, crucial for assessing how well the model can be expected to perform on new, unseen data.
-Correcting the variance calculation by using \( n-1 \) instead of \( n \) compensates for the tendency of smaller samples (specific datasets of \gls{libs} spectral data) to underestimate the variability of the entire population (all possible \gls{libs} data), bringing the sample standard deviation closer to the true standard deviation of the entire population.
 A lower standard deviation indicates a more robust model across different samples.
 
-Success will be evaluated primarily through these metrics, comparing the predictive accuracy and robustness of our models against existing benchmarks and baseline models established in prior research.
+\subsection{Challenges}
+As mentioned, quantifying chemical compositions from \gls{libs} spectral data involves several significant challenges that must be addressed to ensure accurate and robust predictions.
+Several of the following challenges can be addressed through known methodologies, such as sample preprocessing, ensuring the repeatability of observations, and selecting calibration samples that are matrix-matched to specific targets of interest\cite{andersonPostlandingMajorElement2022}.
+However, these strategies are not feasible for the extraterrestrial applications relevant to this research.
 
-\textbf{Problem Definition:} This thesis aims to address the challenges in predicting major oxide compositions from \gls{libs} data by enhancing computational methods to improve accuracy and robustness. We propose to develop computational models capable of effectively accounting for and mitigating the complexities inherent in \gls{libs} data. Our models will take as input a matrix in the form of $E$, as well as ground truth data in the form of $A$, to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_{o}}$, mapping processed \gls{libs} signals to estimated oxide concentrations.
+\subsubsection{Data Dimensionality}
+The large number of dimensions, as seen by having many wavelengths $\lambda$ in the Intensity Matrix $\mathbf{I}[\chi, l, s, \lambda]$, leads to problems such as noise, computational inefficiency, and instability in model predictions.
```

Comment:
I don't think that saying "high dimensionality = noise" is correct - noise is a consequence of the instrumental measurements from what I understand.

Reply:
We say that higher dimensionality in data can lead to noise, so it's not as absolute as `high dimensionality = noise`. And not trying to claim that high dimensionality is the only source of noise, either. Just that it could imply that many features aren't necessary, or perhaps directly affect, predicting the targets.

Reply:
Really hard to get into the source of noise, though. We could mention noise induced by measurement, but then we'd also want to specify that it's already being handled before we even get the data - pre-CCS, that is.